### ISSUE TITLE
l10n_fr_siret: unrequired dependency on account

### ISSUE THREAD

**Issue by gurneyalex** [OCA/l10n-france#38]
It would be great if we could split a bit l10n_fr_siret to remove the dependency on account. 

We have one customer at least who would like to record a SIRET number on partners but does not wish to pay an Enterprise Contract for accounting which he is not going to use.

**Comment by mdietrichc2c** [comment]
It seems that the only dependency on `account` is the partner view, which adds new fields in the "Accounting" tab, so if the basic view is used instead, it should be enough to remove the dependency.

**Comment by charbeljc** [comment]
Right. We should also remove dependency on Odoo's `l10n_fr` as a whole, as it also depends on `account`. In the meantime, I'm just copying the bits we need from `l10n_fr_siret`.

**Comment by alexis-via** [comment]
As explained in this comment:

https://github.com/OCA/l10n-france/blob/8.0/l10n_fr_siret/__openerp__.py#L31

the dependency on account is only for a matter of view. But I think it is justified, because you expect to have the SIRET number in the same tab as the VAT number... I don't see a better place for it in the view.

By the way, I thought that the "price per module" for the Entreprise contrat for something of the past... Even if it's still the case, I don't think it's a good justification to change the module. I think that, for such a particular case, you should fork the module for your customer (or propose a PR on the official addons to re-organise the views... which will certainly never be accepted).

**Comment by charbeljc** [comment]
@alexis-via How about splitting into `l10n_fr_siret_base` and `l10n_fr_siret`, the last one auto-installable and depending on `l10n_fr_siret_base` and `account` ?

**Comment by alexis-via** [comment]
@charbeljc Yes, it's the solution. It will allow you to remove the dependency on 'account'. If you also need to remove the dependency on 'l10n_fr', you need to handle the fact that l10n_fr contains the definition of the 'siret' field (which is a bad thing ; we should remove these fields from 'l10n_fr' and define them in some module in OCA/l10n-france.

**Comment by clonedagain** [comment]
If you split in 2 modules name please them `l10n_fr_siret` and  `l10n_fr_siret_account` to respect the naming convention.
If you want to get rid of the dependency on `l10n_fr` you'd have to also make a glue module `l10n_fr_siret_l10n_fr`.
Please make sure they're auto-installed.

**Comment by charbeljc** [comment]
Hi @clonedagain, ok for naming conventions. For the second point, yes, as `l10n_fr` depends on `account`, we definitively want to get rid of this dependency and define all siret information in `l10n_fr_siret`. And we also want to make sure that installing `l10n_fr` wont override definition given by previous `l10n_fr_siret` installation.

From what I understand, this would be the role of our `l10n_fr_siret_l10n_fr` auto-installable glue module.
Could you elaborate on this (what should be the dependencies ? `l10n_fr_siret` + `l10n_fr` ?)

Thanks you !
